core/templates/pages/exploration-editor-page/exploration-editor-page.component.html

```diff
@@ -35,6 +35,8 @@
   </div>
 </div>
 
+<on-screen-keyboard customizable="false" custom-letters="''"></on-screen-keyboard>
```

Comment:
Instead of adding this here should we add this in the interaction which uses it? (not sure!)

Reply:
Yeah, that would be ideal (in terms of easily exchanging interaction data with the component). But the issue with that is that the OSK needs to be displayed from the bottom of the screen overlaying on top of all other entities...and if we add this in the interactions then it would only be displayed inside the white div designated for the questions and answers. This is why I had to add this somewhere higher up.